components/financial_assistance/spec/shared_examples/medicaid_gateway/Simple_Test_Case_E_New.xml

```diff
@@ -633,4 +634,4 @@
         <ns5:HouseholdMemberReference ns1:ref="pe1992374604681766995"/>
         <ns5:HouseholdMemberReference ns1:ref="pe1992374604681766996"/>
     </ns3:PhysicalHousehold>
-</ns10:AccountTransferRequest>
```

Comment:
Its a whitespace removal not the `AccountTransferReques`t tag, I'm not sure why it shows up in this way.